File: check.js
import { query } from "./db.js";
async function run() {
  const res1 = await query("SELECT column_name, data_type FROM information_schema.columns WHERE table_name = 'event_budgets'");
  console.log("event_budgets:", res1.rows);
  process.exit();
}
run();

File: check.py
import psycopg2
conn = psycopg2.connect('postgresql://postgres.widgcpsntyrzrxalhifa:TheBigBannaBoy123@aws-1-ap-south-1.pooler.supabase.com:6543/postgres')
cur = conn.cursor()
cur.execute("SELECT table_name, column_name FROM information_schema.columns WHERE table_name LIKE 'participant_teams%'")
for row in cur.fetchall():
    print(row)
